operator/scripts/create-olm-bundle.sh

```diff
@@ -69,6 +69,9 @@ yq ea -i '.spec.install.spec.deployments[0].spec.template.metadata.labels.name =
 yq ea -i '.spec.install.spec.deployments[0].spec.template.spec.containers[0].env += [{"name": "POD_NAME", "valueFrom": {"fieldRef": {"fieldPath": "metadata.name"}}}]' "$CSV_PATH"
 yq ea -i '.spec.install.spec.deployments[0].spec.template.spec.containers[0].env += [{"name": "OPERATOR_NAME", "value": "keycloak-operator"}]' "$CSV_PATH"
 
+# Remove ServiceMonitors GVK from nativeAPIS to allow CSV installation when CRDs not present
+yq ea -i 'del(.spec.nativeAPIs[] | select(.kind == "ServiceMonitor")) | del(.spec.install.spec.permissions[].rules[] | select(.resources[] == "servicemonitors"))' "$CSV_PATH"
```

Comment:
Give this is OLM specific, we should probably include `ServiceMonitor` in OLM tests as well.

Reply:
I have updated CI so that we test OLM with and without the ServiceMonitor CRDs present. To reduce repetition, I have added some additional scripts to `operator/scripts`. I also have added some additional steps to capture CR status and the cluster state with `oc adm inspect` when a failure occurs to make it easier to determine issues without recreating everything locally.